src/main/java/org/folio/service/CustomFieldsService.java

```diff
@@ -18,7 +18,7 @@
 
 import io.vertx.core.Future;
 import io.vertx.core.Vertx;
-import io.vertx.core.impl.CompositeFutureImpl;
+import io.vertx.core.impl.future.CompositeFutureImpl;
```

Comment:
Please do not use internal class CompositeFutureImpl, this is not part of the Vert.x API: https://vertx.io/docs/apidocs/
Use https://github.com/folio-org/okapi/blob/master/okapi-common/src/main/java/org/folio/okapi/common/GenericCompositeFuture.java instead.